extension/src/helpers/__tests__/stellar.test.ts

```diff
@@ -0,0 +1,11 @@
+import { truncatedPublicKey } from "../stellar";
+
+describe("truncatedPublicKey", () => {
+  it("truncates keys", () => {
+    expect(
+      truncatedPublicKey(
+        "GAJVUHQV535IYW25XBTWTCUXNHLQN4F2PGIPOOX4DDKL2UPNXUHWU7B3",
+      ),
+    ).toBe("GAJV…U7B3");
```

Comment:
I wanted the demo test to be a react testing library one, but for some reason that's causing the build to fail with linter errors?? Unclear but didn't want to block the PR on that